Microsoft.Toolkit.Uwp.SampleApp/landingPageLinks.json

```diff
@@ -1,6 +1,6 @@
 {
   "new-section-title": "What's New",
-  "new-samples": [ "EyeDropper", "ViewportBehavior", "LoginButton", "PeoplePicker", "PersonView" ],
+  "new-samples": [ "TokenizingTextBox", "PipelineBrush", "WrapLayout", "IsNullOrEmptyStateTrigger", "High Performance APIs" ],
```

Comment:
@skendrot @dotMorten we only have 5 slots and can only link to a specific Sample currently; so I had the community vote on my stream and we picked representative samples from your collections (ItemsRepeater, Triggers). I also felt like they made good introductions/samples to the topics as well.

Do you agree with these choices? If you'd like to flip to another one, let me know here with a code suggestion update.